cogs/vxtwitconverter/vxtwitconverter.py

```diff
@@ -0,0 +1,44 @@
+from redbot.core import commands
+from redbot.core.bot import Red
+from discord import Message
+from urlextract import URLExtract
+
+
+class VxTwitConverter(commands.Cog):
+    """Converts twitter link embeds to vxtwitter"""
+
+    def __init__(self, bot: Red):
+        super().__init__()
+        self.bot = bot
+
+    @commands.Cog.listener("on_message")
+    async def twit_replacer(self, message: Message):
+        # skips if the message is sent by the bot
```

Comment:
Can you please add a check to make sure the message is coming from a server/guild and not a DM? If it's not from a guild, then bail early.